apps/deploy-web/src/components/remote-deploy/api/api.ts

```diff
@@ -0,0 +1,171 @@
+import { useMutation, useQuery } from "react-query";
```

Comment:
We could extract these as services extending the `HttpService` https://github.com/akash-network/console/tree/main/packages/http-sdk/src

Thoughts @ygrishajev ?

Reply:
definitely